Formula/migrate.rb

```diff
@@ -0,0 +1,27 @@
+class Migrate < Formula
```

Comment:
`migrate` is a Homebrew command so this needs a different name.

Reply:
does arbitrary class name affect installation? new to ruby, I'm considering `golang-Migrate` as new class name

Reply:
brew says `Expected to find class Migrate, but only found: GolangMigrate.`, any idea?